<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenImageDialog</class>
 <widget class="QDialog" name="OpenImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenImage</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QPushButton" name="OKButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="FileInfoGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="title">
    <string>FileInfo</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="fileNameLabel">
          <property name="font">
           <font>
            <family>Sans Serif</family>
           </font>
          </property>
          <property name="text">
           <string>File Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="getFileNameLable">
          <property name="font">
           <font>
            <family>Sans Serif</family>
           </font>
          </property>
          <property name="text">
           <string>unknown File</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="imageSizeLabel">
          <property name="font">
           <font>
            <family>Sans Serif</family>
           </font>
          </property>
          <property name="text">
           <string>Image Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="getImageSizeLabel">
          <property name="font">
           <font>
            <family>Sans Serif</family>
           </font>
          </property>
          <property name="text">
           <string>0 Slices, 0 * 0 , 0 bpp</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="ResolutionRoupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
    </font>
   </property>
   <property name="title">
    <string>Resolution</string>
   </property>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>31</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="ResolutionStyleLabel">
       <property name="font">
        <font>
         <family>Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Define : Voxel size (um)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="VoxelSizeLabel">
       <property name="sizeIncrement">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>Voxel Size:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="FilePathDisplayLabel">
      <property name="text">
       <string>Image Path:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FilePathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="OpenButton">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="SomaPathDisplayLabel">
      <property name="text">
       <string>Stack Path:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StackPathEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="OpenStackButton">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OpenImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
